hack/testing/logging.sh

```diff
@@ -491,6 +491,7 @@ os::cmd::expect_success "oc patch $ops_dc \
    -p '{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"elasticsearch\",\"securityContext\":{\"privileged\": true}}]}}}}'"
 
 os::cmd::try_until_text "oc deploy $ops_dc --latest" "Started" "$(( 3 * TIME_MIN ))"
+oc logs -f $ops_dc
```

Comment:
What should we use with `try_until_text`?  Note that `oc deploy` returns immediately - it doesn't really make sense to use `try_until_text` with it, since it will either return with success or error.  Should we use `try_until_text` with the "ephemeral" `-deploy` pod?  That's what `oc logs -f` is looking at in this case.  Should we wait until the es-ops pod is running?  If so, how do we know which es-ops pod corresponds to the `oc deploy` deployment?